lib/fluent/plugin/webhdfs_compressor_zstd.rb

```diff
@@ -0,0 +1,24 @@
+module Fluent::Plugin
+  class WebHDFSOutput < Output
+    class ZstdCompressor < Compressor
+      WebHDFSOutput.register_compressor('zstd', self)
+
+      def initialize(options = {})
+        begin
+          require "zstandard"
+        rescue LoadError
+          raise Fluent::ConfigError, "Install libzstd before use zstd compressor"
```

Comment:
`zstandard gem` instead of `libzstd` is better.

Reply:
fixed in https://github.com/fluent/fluent-plugin-webhdfs/pull/76/commits/cf351b870cb61503343b168509f965593f4c9623